notify/notify.yaml

```diff
@@ -54,6 +110,63 @@ paths:
           application/json:
             schema:
               $ref: '#/components/schemas/create_webhook'
+      x-readme:
+        samples-languages:
+          - javascript
+          - curl
+          - python
+          - go
+        code-samples:
+          - language: javascript
+            name: Alchemy SDK
+            code: |
+              // Setup: npm install alchemy-sdk
+              // Github: https://github.com/alchemyplatform/alchemy-sdk-js
+              import { Alchemy, Network, WebhookType } from "alchemy-sdk";
+
+              // authToken is required to use Notify APIs. Found on the top right corner of
+              // https://dashboard.alchemy.com/notify.
+              const settings = {
+                authToken: "your-notify-auth-token",
+                network: Network.ETH_MAINNET, // Replace with your network.
+              };
+
+              const alchemy = new Alchemy(settings);
+
+              const minedTxWebhook = await alchemy.notify.createWebhook(
+                "https://webhook.site/your-webhook-url",
+                WebhookType.MINED_TRANSACTION,
+                { appId: "wq9fgv022aff81pg" }
```

Comment:
do we want to include the real app id here, or use placeholder text?

Reply:
we can invalidate it - or use "demo"?

Reply:
It shouldn't work as long as we don't expose the auth token, right? I can change some characters to make it impossible to access